.releaserc.js

```diff
@@ -51,5 +51,26 @@ module.exports = {
         packageName: 'forest-express-mongoose',
       }
     ],
+    [
+      "semantic-release-npm-deprecate-old-versions", {
+        "rules": [
+          { 
+            "rule": "supportLatest", 
+            "options": {
+              "numberOfMajorReleases": 2,
```

Comment:
If we agree to support 3 major versions, I find it a bit misleading to read "2", I would expect to read this config:
```suggestion
              "numberOfMajorReleases": 3,
```
to include the current major version.

I guess it is correctly implemented like this with your package, so it is just a "style" comment about the way the configuration is exposed.

Reply:
The package is behaving as you guessed, I just made a mistake because I thought we wanted to support 2 versions